### PR TITLE
Add scopes - batch 1 for Entra PowerShell cmdlets

### PR DESCRIPTION
**Pull request by SteveMutungi254**
In this PR, I have added:
- Scopes for the cmdlet.
- Enriched description with additional information from API.
- Removed command prompt.

### Cmdlets in scope:
- Set-EntraDirSyncConfiguration
- Set-EntraDirSyncFeature
- Set-EntraMSApplication
- Set-EntraMSApplicationLogo
- Add-EntraEnvironment
- Convert-EntraFederatedUser
- Enable-EntraDirectoryRole
- Get-EntraContactDirectReport
- Get-EntraContactThumbnailPhoto
- Get-EntraCurrentSessionInfo
- Get-EntraDirSyncConfiguration
- Get-EntraDirSyncfeature
- Get-EntraAccountSku
- Get-EntraApplicationKeyCredential
- New-EntraApplicationKeyCredential
- New-EntraMSApplication
- Get-EntraUserThumbnailPhoto
- Remove-EntraApplicationKeyCredential
- Remove-EntraMSApplication
- Remove-EntraMSApplicationOwner
- Set-EntraUserThumbnailPhoto
- Update-EntraSignedInUserPassword
- Set-EntraTenantDetail

### CODE REVIEW

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md [hunk 1]
```diff
@@ -64,10 +67,11 @@ This example shows how to retrieve the application logo for an application that
 ## Parameters
 
 ### -FileName
+
 If provided, the application logo is copied to the file who's name is provided in this parameter.
```
Comment (alexandair) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md:71]: If provided, the application logo is saved to the file using the specified file name.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md [hunk 2]
```diff
@@ -79,10 +83,11 @@ Accept wildcard characters: False
 ```
 
 ### -FilePath
+
 If provided, the application logo is copied with a random filename to the file path that is specified in this parameter.
```
Comment (alexandair) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md:87]: If provided, the application logo is saved to the specified file path using a random file name.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md [hunk 3]
```diff
@@ -109,10 +115,11 @@ Accept wildcard characters: False
 ```
 
 ### -View
+
 If set to $true, the application's logo is displayed in a new window on the screen.
```
Comment (alexandair) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md:119]: If set to $true, displays the application logo in a new window.

@SteveMutungi254 Shouldn't this Boolean parameter be a switch parameter?

**Review by alexandair — CHANGES_REQUESTED** [review]

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Convert-EntraFederatedUser.md [hunk 4]
```diff
@@ -75,12 +89,15 @@ Accept wildcard characters: False
...
+
+This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## Inputs
 
 ## Outputs
 
 ## Notes
 
+- See more details : <https://learn.microsoft.com/graph/api/authenticationmethod-resetpassword>
```
Comment (csmulligan) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Convert-EntraFederatedUser.md:101]:
```suggestion
- For more information, see [resetPassword](/graph/api/authenticationmethod-resetpassword).
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Enable-EntraDirectoryRole.md [hunk 5]
```diff
@@ -68,17 +75,19 @@ Accept wildcard characters: False
...
+
+This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## Inputs
 
 ## Outputs
 
 ## Notes
 
+- See additional details - <https://learn.microsoft.com/graph/api/directoryrole-post-directoryroles>
```
Comment (csmulligan) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Enable-EntraDirectoryRole.md:87]:
```suggestion
- For additional details see [Activate directoryRole](/graph/api/directoryrole-post-directoryroles).
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraDirSyncConfiguration.md [hunk 6]
```diff
@@ -109,8 +123,11 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 ## Outputs
 
 ### System.Object
+
 ## Notes
 
+- See additional details - <https://learn.microsoft.com/graph/api/onpremisesdirectorysynchronization-update>
```
Comment (csmulligan) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraDirSyncConfiguration.md:129]:
```suggestion
- For additional details see [Update onPremisesDirectorySynchronization](/graph/api/onpremisesdirectorysynchronization-update).
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraDirSyncFeature.md [hunk 7]
```diff
@@ -137,14 +161,18 @@ Accept wildcard characters: False
...
+
+This cmdlet supports the common parameters: `-Debug`, `-ErrorAction`, `-ErrorVariable`, `-InformationAction`, `-InformationVariable`, `-OutVariable`, `-OutBuffer`, `-PipelineVariable`, `-Verbose`, `-WarningAction`, and `-WarningVariable`. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## Inputs
 
 ## Outputs
 
 ## Notes
 
+- See additional details - <https://learn.microsoft.com/graph/api/onpremisesdirectorysynchronization-update>
```
Comment (csmulligan) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraDirSyncFeature.md:173]:
```suggestion
- For additional details see [Update onPremisesDirectorySynchronization](/graph/api/onpremisesdirectorysynchronization-update).
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraDirSyncFeature.md [hunk 8]
```diff
@@ -137,14 +161,18 @@ Accept wildcard characters: False
...
+- See feature list - <https://learn.microsoft.com/graph/api/resources/onpremisesdirectorysynchronizationfeature>
```
Comment (csmulligan) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraDirSyncFeature.md:174]:
```suggestion
- For the feature list see the [onPremisesDirectorySynchronizationFeature resource type](/graph/api/resources/onpremisesdirectorysynchronizationfeature).
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraTenantDetail.md [hunk 9]
```diff
@@ -151,16 +178,21 @@ Accept wildcard characters: False
...
 
 ### None
+
 ## Outputs
 
 ### System.Object
+
 ## Notes
 
+- See more details - <https://learn.microsoft.com/graph/api/organization-update>
```
Comment (csmulligan) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Set-EntraTenantDetail.md:194]:
```suggestion
- For more details see [Update organization](/graph/api/organization-update).
```

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Update-EntraSignedInUserPassword.md [hunk 10]
```diff
@@ -79,12 +89,15 @@ Accept wildcard characters: False
...
 
 ## Inputs
 
 ## Outputs
 
 ## Notes
 
-## RELATED LINKS
\ No newline at end of file
+- See more details - <https://learn.microsoft.com/graph/api/user-changepassword>
```
Comment (csmulligan) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Update-EntraSignedInUserPassword.md:101]:
```suggestion
- For more details see [changePassword](/graph/api/user-changepassword).
```

**Review by csmulligan — APPROVED** [review]
Hi @SteveMutungi254 I added a few minor suggestions. LGTM, thanks.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md (hunk 1 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md]: Added. Thank you.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md (hunk 3 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md]: Suggestion Added. Thank you.

_re: Boolean parameter:_

@alexandair : I have added this to our backlog to have a global clean-up/implementation.

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md (hunk 2 above)
Comment (SteveMutungi254) [reply module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md]: Suggestion added. Thank you.

**Comment by SteveMutungi254** [comment]
> Hi @SteveMutungi254 I added a few minor suggestions. LGTM, thanks.

Resolved all the comments. Thank you. @csmulligan

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**